### DISCUSSION TITLE
Which address are called by WatchTower?

### DISCUSSION THREAD

**Discussion by Radins** [containrrr/watchtower#1418] [Q&A] [answered]
I'm going to close all outgoing requests from a server, except the ones I'm using, witch includes WatchTower. So i need to kwon all the address WatchTower calls. Probably `https://hub.docker.com` is in this list, but can anyone confirm all the address/DNS used?

**Comment by piksel** [comment, marked as answer]
It does a HEAD request to check for changes for each container image. For docker hub that would result in connections to `index.docker.io` and `auth.docker.io`. So it depends on what registries you are using.

The actual pulls are performed by docker itself (not watchtower) and so they should not be affected.
